webapp/README.md

```diff
@@ -0,0 +1,29 @@
+# Forest Fire Predictions Simple Flask App in Python
+
+#### Simple App Template to deploy and showcase your Forest Fire Machine Learning Prediction Model in Python. Deployed on the server using Flask.
+
+#### [Watch the live demo of the app here](http://kfpro.pythonanywhere.com/forestfirepreds) at kfpro.pythonanywhere.com/forestfirepreds !
+
+#### [Code for the app is available at](https://github.com/khanfarhan10/ForestFirePredictionsFlaskPython) at github.com/khanfarhan10/ForestFirePredictionsFlaskPython !
+
+# Usage
+
+### Fields to Fill in
+
+![Fields](/images/fields.png)
+
+### Working Snapshot of the application
+
+![Working Screenshot](/images/working.png)
+
+# Tutorials
+
+Some great tutorials that I learnt coding for this app are :
+
+- Quick Flask deployment using www.pythonanywhere.com : [Deploying Flask Apps Using Python Anywhere (2020) - by Pretty Printed](https://www.youtube.com/watch?v=5jbdkOlf4cY)
+- Detailed Flask Tutorial : [Python Flask Tutorial: Full-Featured Web App Part 1 - Getting Started - by Corey Schafer](https://www.youtube.com/watch?v=MwZwr5Tvyxo)
+- Useful Tool for writing html files : [W3Schools Try HTML](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_form_checkbox)
+
+###### Note: If the live demo doesn't work anymore, or you need help on the topic feel free to reach out at njrfarhandasilva10@gmail.com.
+
+###### Also, for ML models you might want to change the requirements.txt file and add numpy, pandas, sklearn etc. to it.
```

Comment:
Convert them to project references. Also mention the issues tab of AI-hacktoberfest instead of your personal email.